webapp/components/survey/TaxonomyDetails/TaxonomyDetails.scss

```diff
@@ -5,7 +5,10 @@
   grid-template-rows: 4rem 1fr 3rem;
   align-items: center;
   height: 100%;
+}
 
+.taxonomy__table-container {
+  height: 100%;
```

Comment:
something fishy here :P was that needed?

Reply:
![Captura de pantalla 2020-07-10 a las 13 00 33](https://user-images.githubusercontent.com/11272068/87147688-589c7200-c2ad-11ea-8220-7a84bd7bc019.png)
:/ i think that yes

Reply:
maybe as we use grid we can apply `align-self: stretch;`what do you think?